docs/processes/release/tasks/versions.md

```diff
@@ -0,0 +1,278 @@
+UNICODE_2_1_5 = getInstance(2, 1, 5, 0);
+UNICODE_2_1_8 = getInstance(2, 1, 8, 0);
+UNICODE_2_1_9 = getInstance(2, 1, 9, 0);
+UNICODE_3_0 = getInstance(3, 0, 0, 0);
+UNICODE_3_0_1 = getInstance(3, 0, 1, 0);
+UNICODE_3_1_0 = getInstance(3, 1, 0, 0);
+UNICODE_3_1_1 = getInstance(3, 1, 1, 0);
+UNICODE_3_2 = getInstance(3, 2, 0, 0);
+UNICODE_4_0 = getInstance(4, 0, 0, 0);
+UNICODE_4_0_1 = getInstance(4, 0, 1, 0);
+UNICODE_4_1 = getInstance(4, 1, 0, 0);
+UNICODE_5_0 = getInstance(5, 0, 0, 0);
+UNICODE_5_1 = getInstance(5, 1, 0, 0);
+UNICODE_5_2 = getInstance(5, 2, 0, 0);
+UNICODE_6_0 = getInstance(6, 0, 0, 0);
+UNICODE_6_1 = getInstance(6, 1, 0, 0);
+UNICODE_6_2 = getInstance(6, 2, 0, 0);
+UNICODE_6_3 = getInstance(6, 3, 0, 0);
+UNICODE_7_0 = getInstance(7, 0, 0, 0);
+ICU_VERSION = getInstance(54, 1, 0, 0);
+ICU_DATA_VERSION = getInstance(54, 1, 0, 0);
+UNICODE_VERSION = UNICODE_7_0;
+UCOL_RUNTIME_VERSION = getInstance(8);
+UCOL_BUILDER_VERSION = getInstance(9);
+UCOL_TAILORINGS_VERSION = getInstance(1);
+}
+In the same file, starting at line 164 (as of 54.1) -
+/\*\*
+\* Data version string for ICU's internal data.
+\* Used for appending to data path (e.g. icudt43b)
+\* @internal
+\* @deprecated This API is ICU internal only.
+\*/
+@Deprecated
+public static final String ICU_DATA_VERSION_PATH = "54b";
+4. [icu4j/pom.xml](http://source.icu-project.org/repos/icu/trunk/icu4j/pom.xml)
+(before ICU4J 60 only)
+<groupId>com.ibm.icu</groupId>
+<artifactId>icu4j</artifactId>
+<version>**55-SNAPSHOT**</version>
+<name>ICU4J</name>
+Only for the final release (including maintenance release), update the <version>
+item to the actual release version (e.g. "54.1", "4.8.1") Otherwise, use (next
+ver)-SNAPSHOT. (e.g. "55-SNAPSHOT").
+5. Time Bombs (before ICU4J 52)
+
+***Note: We no longer use time bombs since ICU4J 52. In the trunk,
+logKnownIssue() is used for skipping known test failures. The new scheme no
+longer depends on the current ICU4J version, so there are no updates necessary
+in test codes when version number is updated. See [Skipping Known Test
+Failures](../../../setup/eclipse/time.md) for more details.***
+
+There might be some test cases intentionally skipped for the current ICU4J
+version. When ICU4J version is updated, these time bombed test cases may show
+up. In this case, you should:
+
+*   Inform the list of failing test cases because of the version change to
+    icu-core ML - ask if someone has plan to fix them.
+*   File a ticket if a time bomb does not have any corresponding trac tikect and
+    put the ticket number as comment
+*   Move the time bomb version to the next milestone.
+
+The time bomb looks like below -
+if (**isICUVersionBefore(49, 1)**) { // ticket#6806
+logln(showOrderComparison(uLocale, style1, style2, order1, order2));
+} else {
+errln(showOrderComparison(uLocale, style1, style2, order1, order2));
+}
+Note: ICU4J time bomb - Before
+[#7973](http://bugs.icu-project.org/trac/ticket/7973), we used to use
```

Comment:
Please fix links to ICU tickets --> Jira

Reply:
Done.